packages/jsii-pacmak/lib/targets/golang/types/class.ts

```diff
@@ -1,29 +1,78 @@
-import { Method, ClassType } from 'jsii-reflect';
+import { Method, ClassType, Initializer } from 'jsii-reflect';
 import { CodeMaker } from 'codemaker';
 import { GoTypeRef } from './go-type-reference';
 import { GoStruct } from './go-type';
 import { TypeField } from './type-field';
+import { getFieldDependencies, substituteReservedWords } from '../util';
 import { Package } from '../package';
+import { ClassConstructor, MethodCall } from '../runtime';
+
+export class GoClassConstructor {
```

Comment:
Would it make sense to have this and `ClassMethod` share a parent class, or have one extend the other?

Reply:
In the future it might. I don't see a ton of similarities right now to extract.